### PR TITLE
AUS ticker switch

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What are you doing in this PR?
- Adds the AUS ticker switch
- Created the ticker switch in the `CODE` and `PROD` dashboards
- adds some documentation around why and how to add a campaign switch 

[**Trello Card**](https://trello.com/c/xD8wc3u2/1704-create-aus-ticker-switch)

## Why are you doing this?
To allow us to do the work for the AUS ticker, merge it to `PROD`, test it, but not have it released to the public.

### CODE REVIEW

File: support-frontend/README.md [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
+To test in code, you would need to follow step 1. on
+[the CODE switchboard](https://support.code.dev-gutools.co.uk/switches).
+
+To test locally, copy the `switches_v2.json` file stored in S3 to the specified path in
+[`DEV.public.conf`](./conf/DEV.public.conf) ie. `~/.gu/support-admin-console/switches_v2.json`.
+
+```bash
+# Make sure you have Janus credentials and `~/.gu/support-admin-console` exists
+aws s3 cp s3://$SUPPORT_ADMIN_BUCKET/ENV/switches_v2.json  ~/.gu/support-admin-console/switches_v2.json --profile membership
+```
```
Comment (jamesgorrie) [inline support-frontend/README.md:46]: I am more than happy to export this to the Wiki if people think it should live there instead.

Comment (rBangay) [reply support-frontend/README.md:46]: Hey, this is great, thanks for doing it ... I'd like to add my vote for putting it in the wiki ... although maybe at some point a contents list of links to the most used/helpful stuff in the wiki might be nice in here.

Comment (jamesgorrie) [reply support-frontend/README.md:46]: Its definitely worth having a think about how we manage our knowledge. The main reason this felt useful here is it's a code thing, and so when I am in my IDE, I can just reference the markdown. I can also 

The wiki itself is also very out of date in some instances, making it hard to know what information to follow and what not to, which I have seen often happen when it is decoupled from the code where it's relevant. 

The other piece is I would, as the author, really appreciate if these instructions are easy to follow, and a PR is a great place for this, so I might leave the content in here until people have reviewed and validated it's good. I am not sure what a good process for this is in the Wiki?

I can then remove it and push it to the wiki.

Comment (jamesgorrie) [reply support-frontend/README.md:46]: I've added it here for now:
https://github.com/guardian/support-frontend/wiki/Quickstart:-Campaign-feature-development

**Review by LAKSHMIRPILLAI — APPROVED** [review]
👍🏼

File: support-frontend/README.md (hunk 1 above)
Comment (jamesgorrie) [reply support-frontend/README.md]: I've removed this from here for now. I think we should whack some Wiki links into the main `README`s as they do seem like the main source of information, but are not massively discoverable from the codebase.

**Pull request merged**